readme-vars.yml

```diff
@@ -46,6 +46,8 @@ opt_param_volumes:
   - { vol_path: "/media", vol_host_path: "</path/to/other media>", desc: "Location of other media." }
 opt_param_usage_include_ports: false
 opt_param_device_map: false
+opt_param_devices:
+  - { device_path: "/dev/snd", device_host_path: "/dev/snd", desc: "Only needed to pass your host sound device to Airsonic's Java jukebox player." }
```

Comment:
All the readme changes above need to be added to this file. The readme.md file is overwritten add generated from this file, so if it's not here it will be lost.

Reply:
Sorry, I don't see how to do that.. Is there a way to try out the readme generation locally? I see in the Jenkinsfile that the `readme-vars.yml` make their way into the release notes, but not the readme...

Reply:
This generates all templated files including the readme: https://github.com/linuxserver/docker-jenkins-builder

Reply:
Thanks, that's what I needed! Should be fixed now.